ethminer/MinerAux.h

```diff
@@ -856,31 +856,24 @@ class MinerCLI
 					}
 					this_thread::sleep_for(chrono::milliseconds(_recheckPeriod));
 				}
-				if (EthashAux::eval(solution.seedHash, solution.headerHash, solution.nonce).value < solution.boundary)
-				{
-					bool ok = prpc->eth_submitWork("0x" + toHex(solution.nonce), "0x" + toString(solution.headerHash), "0x" + toString(solution.mixHash));
-					if (ok) {
-						cnote << "Solution found; Submitted to" << _remote << "...";
-						cnote << "  Nonce:" << solution.nonce;
-						cnote << "  headerHash:" << solution.headerHash.hex();
-						cnote << "  mixHash:" << solution.mixHash.hex();
-						cnote << EthLime << "Accepted." << EthReset;
-						f.acceptedSolution(false);
-					}
-					else {
-						cwarn << "Solution found; Submitted to" << _remote << "...";
-						cwarn << "  Nonce:" << solution.nonce;
-						cwarn << "  headerHash:" << solution.headerHash.hex();
-						cwarn << "  mixHash:" << solution.mixHash.hex();
-						cwarn << "Not accepted.";
-						f.rejectedSolution(false);
-					}
-					//exit(0);
+				bool ok = prpc->eth_submitWork("0x" + toHex(solution.nonce), "0x" + toString(solution.headerHash), "0x" + toString(solution.mixHash));
+				if (ok) {
+					cnote << "Solution found; Submitted to" << _remote << "...";
+					cnote << "  Nonce:" << solution.nonce;
+					cnote << "  headerHash:" << solution.headerHash.hex();
+					cnote << "  mixHash:" << solution.mixHash.hex();
+					cnote << EthLime << "Accepted." << EthReset;
+					f.acceptedSolution(false);
 				}
 				else {
-					f.failedSolution();
-					cwarn << "FAILURE: GPU gave incorrect result!";
+					cwarn << "Solution found; Submitted to" << _remote << "...";
+					cwarn << "  Nonce:" << solution.nonce;
+					cwarn << "  headerHash:" << solution.headerHash.hex();
+					cwarn << "  mixHash:" << solution.mixHash.hex();
+					cwarn << "Not accepted.";
+					f.rejectedSolution(false);
```

Comment:
same here ``f. rejectedSolution(solution.stale);`` would be better, since we now have it